sw/device/tests/BUILD

```diff
@@ -2535,6 +2535,40 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "spi_device_tpm_tx_rx_test",
+    srcs = ["spi_device_tpm_tx_rx_test.c"],
+    cw310 = new_cw310_params(
+        # This test requires the spi full duplex on the hyperdebug board.
+        tags = ["manual"]
+        test_cmd = """
+            --bitstream="{bitstream}"
+            --bootstrap="{firmware}"
+            "{firmware:elf}"
+        """,
+        test_harness = "//sw/host/tests/chip/spi_device_tpm_test",
+    ),
+    exec_env = {
+        "//hw/top_earlgrey:fpga_cw310_sival": None,
+        "//hw/top_earlgrey:fpga_cw310_test_rom": None,
```

Comment:
Is this test compatible with the SAM3X variant? I don't think we have IOA7 connected for the chip select. Might have to delete this line.